.github/workflows/ci.yml

```diff
@@ -167,6 +167,14 @@ jobs:
           pnpm install --frozen-lockfile
           pnpm playwright install --with-deps
 
+      - name: Setup Bun
```

Comment:
If this fixes the pipeline somehow, we should add a TODO to look into it more and remove it. Bun should not even be needed, and this will add some time to CI.